### ISSUE TITLE
v.1.4 options not being saved

### ISSUE THREAD

**Issue by JSZAdmin** [barry-ran/QtScrcpy#224]
hi glad to see that some of Scrcpy 1.14 was sync'ed :)

The at leased two of the added options from [feat: sync scrcpy](https://github.com/barry-ran/QtScrcpy/commit/624337157a07a212916ac2c5c45a4180b7af704a) does not seem to be saved

lockVideoOrientation
stayAwake

Whatever i set them to they simply reset to default values when QtScrcpy is restarted

**Comment by masterchiang** [comment]
you are right,original version has been supported save the window location with long short

**Comment by barry-ran** [comment]
> hi glad to see that some of Scrcpy 1.14 was sync'ed :)
> 
> At leased two of the added options from [feat: sync scrcpy](https://github.com/barry-ran/QtScrcpy/commit/624337157a07a212916ac2c5c45a4180b7af704a) does not seem to be saved
> 
> lockVideoOrientation
> stayAwake
> 
> Whatever i set them to they simply reset to default values when QtScrcpy is restarted

Yes, I know,  There will be a major revision of UI, I will add lockVideoOrientation&stayAwake to config later
